scripts/README.md

```diff
@@ -0,0 +1,20 @@
+# Shotgun Pipeline Toolkit Core API
+
+## How to upgrade pyyaml
```

Comment:
This file and the upgrade script should go into the `developer` folder, where we also have the `shotgun_api_update.sh` script